patches/coreboot-talos_2_support_payload/0004-Makefile.inc-directly-call-create-container.patch

```diff
@@ -0,0 +1,77 @@
+From 514ad8f2a486d5efcb28d8e6f487a4afc7735e9d Mon Sep 17 00:00:00 2001
+From: Sergii Dmytruk <sergii.dmytruk@3mdeb.com>
+Date: Tue, 6 Jul 2021 18:22:33 +0300
+Subject: [PATCH 4/6] Makefile.inc: directly call create-container
+
```

Comment:
I believe we could use explanation for this patch. And again: fixable in main repo, rather than patch?